packages/yuudachi/locales/en-US/translation.json

```diff
@@ -717,8 +717,16 @@
 				}
 			},
 			"table": {
-				"success_titles": ["Case id", "Member id", "Username"],
-				"fail_titles": ["Member id", "Username", "Error"]
+				"success_titles": [
+					"Case id",
+					"Member id",
+					"Username"
+				],
+				"fail_titles": [
+					"Member id",
+					"Username",
+					"Error"
+				]
```

Comment:
how did formatting change here? thought we use the same config

Reply:
IDK, my fucking eslint does this
Maybe its bc I have it installed globally, hmmmm

Reply:
Fixed in 5549865